hw2/utils.ts

```diff
@@ -0,0 +1,72 @@
+import { User } from './models/user.model';
+
+let usersList: User[] = [
```

Comment:
```
const usersList: User[] = [
```

Reply:
`const` only blocks reassign of an variable(`a = ...`). So you could still use it with objects and arrays.